content/faqs/How-do-I-uninstall-remove-Privacy-Badger.es.md

```diff
@@ -3,8 +3,10 @@ question: ¿Cómo desinstalo o elimino Privacy Badger?
 weight: 21
 ---
 
-Firefox: Vea la página ["Cómo eliminar extensiones y temas"](https://support.mozilla.org/en-US/kb/disable-or-remove-add-ons#w_how-to-remove-extensions-and-themes) de Soporte de Mozilla.
+Firefox: Consulte la página de ayuda [Desinstalar o eliminar los complementos](https://support.mozilla.org/es/kb/desinstalar-o-eliminar-los-complementos#w_eliminar-extensiones-y-temas) de Mozilla.
 
-Cromo: Consulte la página de ayuda de ["Install and manage extensions"](https://support.google.com/chrome_webstore/answer/2664769?hl=en) Chrome Web Store.
+Cromo: Consulte la página de ayuda [Instalar y administrar extensiones](https://support.google.com/chrome_webstore/answer/2664769?hl=es) de Chrome Web Store.
 
-Ópera: Haga clic en el botón del menú en la parte superior izquierda de la ventana y, a continuación, haga clic en "Extensiones" y en "Gestionar extensiones". Desplácese hasta que vea Privacy Badger, mueva el ratón sobre él y, a continuación, haga clic en el icono "X" en la parte superior derecha. Haga clic en "Aceptar" para confirmar la eliminación. A continuación, puede cerrar de forma segura la ficha Extensiones.
\ No newline at end of file
+Edge: Consulte la página de ayuda [Agregar o quitar complementos, extensiones y barras de herramientas del navegador](https://support.microsoft.com/es-mx/help/4027935/microsoft-edge-add-or-remove-browser-extensions) de Microsoft.
+
+Ópera: Haga clic en el botón del menú en la parte superior izquierda de la ventana y, a continuación, haga clic en "Extensiones" y en "Gestionar extensiones". Desplácese hasta que vea Privacy Badger, mueva el ratón sobre él y, a continuación, haga clic en el icono "X" en la parte superior derecha. Haga clic en "Aceptar" para confirmar la eliminación. A continuación, puede cerrar de forma segura la ficha Extensiones.
```

Comment:
Similarly, it seems that Opera itself uses "Opera", not "Ópera", in their Spanish locale.